web_poet/page_inputs.py

```diff
@@ -74,6 +75,45 @@ def from_name_value_pairs(cls: Type[T_headers], arg: List[Dict]) -> T_headers:
         """
         return cls([(pair["name"], pair["value"]) for pair in arg])
 
+    @classmethod
+    def from_bytes(
```

Comment:
The name may be a bit misleading, one may think this creates Headers from HTTP response bytes which represent the headers. Maybe something like `from_bytes_dict`? Or maybe even more specific, `from_scrapy_headers`?

Reply:
Good point! Renaming it to `from_scrapy_headers` seems to be too specific. Other non-scrapy libs could have the same setup. Having it as `from_bytes_dict` makes it more clear and semantically more meaningful at the same time. Updated this on https://github.com/scrapinghub/web-poet/pull/33/commits/eb184270ff8db4bccbb7e3f7317c0192a0fb6a29